features/_your_details/your_details_disabling_minimum_contributions.feature

```diff
@@ -6,12 +6,12 @@ Feature: Your Details disabling minimum contribution
   @no-javascript
   Scenario Outline: Annual salary rate below £5,876 with minimum employer contributions
     Given I am on the Your Details step
-    When I enter my details
+    And I enter my details
```

Comment:
I like the fact that we are now starting to list contexts under the _Given_ keyword. 

We need to get all other scenarios which doesn't follow this pattern to follow suit. 

```
Scenarios all follow the same pattern: 

1. _Given_ ... Get the system into particular state.
2. _When_ ... Poke it 
3. _Then_ ... Examine the new state
```